Scavenge-UW/Scavenge
# Implement routing, navbar, and inventory view

**How to Test**
1. Login with a valid crendential
2. The left section of the Navbar should be changed from
![image](https://user-images.githubusercontent.com/13430641/112400225-e3a38800-8cd5-11eb-9d92-02cd6ac61f2d.png)
to
![image](https://user-images.githubusercontent.com/13430641/112400204-d8505c80-8cd5-11eb-857b-0ab051efe787.png)
3. Click Manage Pantry
4. PantryAdminView should be shown like below:
![image](https://user-images.githubusercontent.com/13430641/112400255-fa49df00-8cd5-11eb-88b6-52a0bf492dc8.png)


**Change Summary**
1. Implements basic routing
2. Creates a basic navbar and changes menu based on user login status. (Does not differentiate civilian user/pantry admin yet due to unimplemented API)
3. Implements Inventory View. Fetches inventory from the server.


**TODO**
1. Show different menu for civilian users and pantry admin
2. Interact with the API to add/update/delete items in the inventory
